src/Pages/Home/Home.jsx

```diff
@@ -0,0 +1,72 @@
+import { useEffect, useState } from 'react';
+import findMyLocation from '../../Helpers/geolocation';
+import endpoints from '../../Helpers/endpoints';
+import { requestToken, instance } from '../../DataService/dataService';
+import { COOKIE } from '../../DataService/cookieService';
+import { CurrWeatherCard } from '../../Components';
+
+import styles from './Home.module.scss';
+
+export default function Home() {
+    const [currWeather, setCurrWeather] = useState(null);
+    const [coords, setCoords] = useState(null);
+    const [locationData, setLocationData] = useState(null);
+    const [isLoading, setIsLoading] = useState(false);
+
+    async function getCurrentWeather() {
+        if (!coords) {
+            return;
+        }
+        try {
+            setIsLoading(true);
+            const { data } = await instance.get(endpoints.CURR_WEATHER(coords));
+            setCurrWeather(data);
+            setIsLoading(false);
```

Comment:
will user see infinite "loading..." in case the request failed?

Reply:
done